### PR TITLE
Generated models and request builders using Typewriter

### PR DESCRIPTION
**Pull request by MIchaelMainer**

### CODE REVIEW

File: src/Microsoft.Graph/Models/Generated/GiphyRatingType.cs [hunk 1]
```diff
@@ -20,14 +20,14 @@ public enum GiphyRatingType
     {
     
         /// <summary>
-        /// Moderate
```
Comment (darrelmiller) [inline src/Microsoft.Graph/Models/Generated/GiphyRatingType.cs]: Any idea how this happened?  @MIchaelMainer

File: src/Microsoft.Graph/Models/Generated/Organization.cs [hunk 2]
```diff
@@ -72,12 +72,6 @@ public Organization()
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "createdDateTime", Required = Newtonsoft.Json.Required.Default)]
         public DateTimeOffset? CreatedDateTime { get; set; }
     
-        /// <summary>
-        /// Gets or sets creation type.
-        /// </summary>
-        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "creationType", Required = Newtonsoft.Json.Required.Default)]
-        public string CreationType { get; set; }
```
Comment (darrelmiller) [inline src/Microsoft.Graph/Models/Generated/Organization.cs]: We need to investigate why this was removed.

File: src/Microsoft.Graph/Models/Generated/TeamArchiveRequestBody.cs [hunk 3]
```diff
@@ -26,7 +26,7 @@ public partial class TeamArchiveRequestBody
         /// Gets or sets ShouldSetSpoSiteReadOnlyForMembers.
         /// </summary>
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "shouldSetSpoSiteReadOnlyForMembers", Required = Newtonsoft.Json.Required.Default)]
-        public bool? ShouldSetSpoSiteReadOnlyForMembers { get; set; }
```
Comment (darrelmiller) [inline src/Microsoft.Graph/Models/Generated/TeamArchiveRequestBody.cs]: And this

File: src/Microsoft.Graph/Requests/Generated/IUserJoinedTeamsCollectionRequest.cs [hunk 4]
```diff
@@ -60,7 +60,7 @@ public partial interface IUserJoinedTeamsCollectionRequest : IBaseRequest
         /// </summary>
         /// <param name="expandExpression">The expression from which to calculate the expand value.</param>
         /// <returns>The request object to send.</returns>
-        IUserJoinedTeamsCollectionRequest Expand(Expression<Func<Group, object>> expandExpression);
```
Comment (darrelmiller) [inline src/Microsoft.Graph/Requests/Generated/IUserJoinedTeamsCollectionRequest.cs]: And this.

File: src/Microsoft.Graph/Requests/Generated/TeamArchiveRequestBuilder.cs [hunk 5]
```diff
@@ -27,10 +27,10 @@ public partial class TeamArchiveRequestBuilder : BaseActionMethodRequestBuilder<
         public TeamArchiveRequestBuilder(
             string requestUrl,
             IBaseClient client,
-            bool? shouldSetSpoSiteReadOnlyForMembers)
+            bool shouldSetSpoSiteReadOnlyForMembers)
             : base(requestUrl, client)
         {
-            this.SetParameter("shouldSetSpoSiteReadOnlyForMembers", shouldSetSpoSiteReadOnlyForMembers, true);
```
Comment (darrelmiller) [inline src/Microsoft.Graph/Requests/Generated/TeamArchiveRequestBuilder.cs]: And this.

File: src/Microsoft.Graph/Models/Generated/GiphyRatingType.cs (hunk 1 above)
Comment (MIchaelMainer) [reply src/Microsoft.Graph/Models/Generated/GiphyRatingType.cs]: This was a change in the latest metadata update on Tuesday.
https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5

File: src/Microsoft.Graph/Requests/Generated/IUserJoinedTeamsCollectionRequest.cs (hunk 4 above)
Comment (MIchaelMainer) [reply src/Microsoft.Graph/Requests/Generated/IUserJoinedTeamsCollectionRequest.cs]: https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5#diff-e34459d76b3a635d414d3549ff656c4cL1699

File: src/Microsoft.Graph/Requests/Generated/TeamArchiveRequestBuilder.cs (hunk 5 above)
Comment (MIchaelMainer) [reply src/Microsoft.Graph/Requests/Generated/TeamArchiveRequestBuilder.cs]: https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5#diff-e34459d76b3a635d414d3549ff656c4cL10530

File: src/Microsoft.Graph/Models/Generated/TeamArchiveRequestBody.cs (hunk 3 above)
Comment (MIchaelMainer) [reply src/Microsoft.Graph/Models/Generated/TeamArchiveRequestBody.cs]: https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5#diff-e34459d76b3a635d414d3549ff656c4cL10530

**Pull request closed without merging**